components/Report/interfaces.rst

```diff
@@ -0,0 +1,68 @@
+Interfaces
+==========
+
+Model Interfaces
+----------------
+
+.. _component_report_model_report-interface:
+
+ReportInterface
+~~~~~~~~~~~~~~~
+
+This interface should be implemented by model representing a single Report.
+
+.. note::
+    For more detailed information go to `Sylius API ReportInterface`_.
+
+.. _Sylius API ReportInterface: http://api.sylius.org/Sylius/Component/Report/Model/ReportInterface.html
+
+Service Interfaces
+------------------
+
+.. _component_report_data-fetcher_data-fetcher-interface:
+
+DataFetcherInterface
+~~~~~~~~~~~~~~~~~~~~
+
+This interface should be implemented by service representing data fetcher.
```

Comment:
Responsible for providing specific data for a report, based on its configuration.